src/main/java/com/bina/varsim/types/VCFInfo.java

```diff
@@ -14,21 +16,46 @@
 public class VCFInfo {
     final private Map<String, VCFInfoElement> info2Value;
 
+    /**
+     * compare speed of different string splitting methods
```

Comment:
Do we know the outcome of this benchmark?

Reply:
just added (for one repeat)